### PR TITLE
Fixing typos and wrong content

### PR DESCRIPTION
**Pull request by deining**

### CODE REVIEW

File: userguide/content/en/docs/Adding content/content.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ title: "Adding Content"
 linkTitle: "Adding Content"
 weight: 1
 description: >
-  How to add content to your Docsy site.
```
Comment (LisaFC) [inline userguide/content/en/docs/Adding content/content.md:107]: I'd leave this as is, I don't think your change makes the description any clearer.

**Review by LisaFC — COMMENTED** [review]
Thanks for the typo fixes! I think the description for the section is clear "as is" and doesn't need changing.

File: userguide/content/en/docs/Adding content/content.md (hunk 1 above)
Comment (deining) [reply userguide/content/en/docs/Adding content/content.md:107]: The motivation of my change is **not** to make the description any clearer.
Have a look at the context and you will understand why I proposed the change:

```
Page frontmatter

...

For example, here’s the frontmatter for this page:

---
title: "Adding Content"
linkTitle: "Adding Content"
weight: 1
description: >
    How to add content to your Docsy site.
---
```

This is simply wrong: the frontmatter of the page `content.md` that is referred to is:

```
---
title: "Adding Content"
linkTitle: "Adding Content"
weight: 1
description: >
  Add different types of content to your Docsy site.
---
```

I guess the description was changed at some point in the past and this needs to be adjusted IMHO.
I hope that makes things clearer.

Comment (LisaFC) [reply userguide/content/en/docs/Adding content/content.md:107]: Aha, sorry, I only looked at the changed section and didn't clock that it was an example within the page. That's fine!

**Review by LisaFC — APPROVED** [review]

**Pull request merged**